src/modules/addRemove.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
- [ ] Please do not disable linters. Linters help us write better code and maintain a uniform coding standard 👍🏾